_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+title: OTel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 72
+---
+
+# OTel metrics raw
+
+The OTel metrics raw processor serializes a collection of `ExportMetricsServiceRequest` sent from the [OTel metrics source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
```

Comment:
```suggestion
The `otel_metrics_raw_processor` serializes a collection of `ExportMetricsServiceRequest` sent from the [OTel metrics source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
```

Reply:
@Naarcha-AWS @natebower Should we include "processor" in the tic marks? Or right after it?

Reply:
The text in tic marks should reflect the name of the processor as it will appear to the user.